### ISSUE TITLE
Can not update SwiftMonkeyPaws 

### ISSUE THREAD

**Issue by upeugene** [zalando/SwiftMonkey#52]
Can not update SwiftMonkeyPaws to 2.0.0 version from CocoaPods.
It shows that last version is 1.0.0.

**Comment by wojciechczerski** [comment]
Hello @hoppus! We should soon push an update to Cocoapods repository (@DagAgren, could you help here?).
In the mean time to use SwiftMonkeyPaws 2.0 with Swift 4 you can use the following entry in your Podfile:

```
pod 'SwiftMonkeyPaws', :git => 'https://github.com/zalando/SwiftMonkey', :tag => '2.0.0'
```

I will give you an update once the new version is pushed.

**Comment by wojciechczerski** [comment]
@hoppus it took more than expected (sorry for that) but I have finally released **SwiftMonkeyPaws 2.0**.
